client/package.json

```diff
@@ -26,7 +26,8 @@
     "three": "^0.141.0",
     "universal-cookie": "^4.0.4",
     "web-vitals": "^2.1.4",
-    "web3.storage": "^4.3.0"
+    "web3.storage": "^4.3.0",
+    "winston": "^3.8.2"
```

Comment:
What's the use of this in the client?

Reply:
I remembered doing that on the server project. But I removed it because it would be on a separate issue will work on later. I do not know how it happened to be there. 
I will remove it and commit back.

As I suggested before we can consider separating the client and server to their respective repo. Maintaining and managing the project will be easy and isolated.

Thanks.

Reply:
Yes, we will separate the client and the server when this hacktoberfest ends.

Thanks.